<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 190, 11, 255), stop:1 rgba(251, 86, 7, 255));
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>처음으로</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;C:/Users/playdata/Desktop/test/pokemon.png&quot;);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-9</y>
     <width>901</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(&quot;C:/Users/playdata/Desktop/test/map1.jpg&quot;);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 170, 200, 200), stop:1 rgba(251, 86, 7, 255));
	border-radius: 12px;
</string>
   </property>
   <property name="text">
    <string>헤어 월드컵</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>graphicsView_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>graphicsView</zorder>
  <zorder>graphicsView_3</zorder>
  <zorder>graphicsView_4</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
